content/general/concepts/api/api.md

```diff
@@ -27,3 +27,25 @@ APIs can be divided into three groups:
 - Private APIs, also known as Internal APIs, are used primarily within a company to share resources and facilitate the business. (e.g. Company Warehouse API for managing inventory with code).
 
 - Partner APIs require rights or specific licenses for use. These APIs are popular in software-as-a-service platforms (e.g. [AWS API](https://docs.aws.amazon.com/general/latest/gr/aws-apis.html)).
+
+## Benefits and Risks of Using APIs
+
+The main benefits of using APIs are:
+
+- Cost saving through the elimination of the need to build and maintain local physical infrastructure and services.
+
+- Opportunity to leverage the expertise of different vendors instead of having to reinvent from scratch things like authentication, payment processing, maps, etc.
```

Comment:
```suggestion
- Opportunity to leverage the expertise of other vendors instead of having to reinvent solutions for common problems. (authentication, payment processing, maps, etc.)
```

Reply:
OK!

Reply:
If this is ok with you, go ahead and click "Commit suggestion", then click "Commit changes".